spec/controllers/v0/institutions_controller_spec.rb

```diff
@@ -40,15 +44,19 @@
   context 'autocomplete results' do
```

Comment:
Within this `context 'autocomplete results'` it would be good to show that autocomplete does not return results of non-approved institutions, like so:
```suggestion
  context 'autocomplete results' do
    it 'does not return results for non-approved institutions' do
      create(:version, :production)
      create(:institution, :contains_harv, approved: false)
      get :autocomplete, term: 'harv', version: 'production'
      expect(JSON.parse(response.body)['data'].count).to eq(0)
      expect(response.content_type).to eq('application/json')
      expect(response).to match_response_schema('autocomplete')
  end
```